File: sum_of_numbers_from_0_to_N_7kyu.py
def show_sequence(n):
    result = str(n) + "<0"
    sum = 0
    
    if n == 0:
        result = "0=0"
    elif(n > 0):
        result = ""
        for i in range(0, n+1):
            sum += i
            result += "+" + str(i)
            
        result = result[1:] + " = " + str(sum)
    
    return result

File: sum_of_numbers_from_0_to_N_7kyu.js
var SequenceSum = (function() {
    function SequenceSum() {}
  
    SequenceSum.showSequence = function(count) {
      var partial = 0;
       var index = 1;
       
      if(count < 0) {
        result = count + '<0';
      } else {
        if(count > 0) {
          result = '0';  
          while(index <= count) {
            result += '+' + index;
            partial += index;
            index++;
          };
          result += ' = ' + partial;
        } else {
          result = '0=0';
        };
      };
       
      return result;
    };
  
    return SequenceSum;
  
  })();
